### ISSUE TITLE
JAWS does not announce the HTML <mark> attribute without CSS included

### ISSUE THREAD

**Issue by Ewallye** [FreedomScientific/VFO-standards-support#548]
We have found that using JAWS 2020 and 2021 that the HTML <mark> attribute is not announcing as expected unless the CSS is included from the following example from Steve Faulkner | December 5, 2017 (https://www.tpgi.com/short-note-on-making-your-mark-more-accessible/) is used. 

NVDA announces the attribute as "marked content" with no added CSS to assist.

### Expected result

When using only the <mark> HTML attribute the marked content would announce some sort of "marked" announcement to provide context to the user using the screen reader.

Is there a way to add the same support as done with NVDA for the <mark> attribute?

### Actual result

It does not announce when only using the <mark> attribute.  The CSS must be added from the URL provided in the summary for JAWS to announce it effectively.  

### Example

https://www.w3schools.com/tags/tag_mark.asp
https://cdpn.io/pavanigutta/debug/OJpYKgp/PBMNWRwJzKxr

#### JAWS version and build number
2020 and 2021 (latest builds)

 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome and Edge

**Comment by StephenBeckettTPGi** [comment]
Behaviour confirmed

**Comment by anant799** [comment]
Faced the same issue.

Operating System and version
MacOS (10.15.7)

Even giving a role as role="mark" is not identifying.
Till when can we expect the issue resolved.

**Comment by voipworld** [comment]
i want to disable jaws reading of html markers...
